recipes/uwsift/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "uwsift" %}
+{% set version = "1.1.0a1" %}
```

Comment:
conda-forge generally does not support a dev release. Is there another version?

Reply:
There are older versions of the library and project but they are not compatible with conda-forge's current offerings. I am hoping to get a release out in the next couple weeks to a month and was hoping to have beta testers walk through the installation instructions using conda-forge.

If this is really not traditionally done then I will find a better way until a full release is out.

NOTE: I would like to learn about options for "dev" labels for a package on conda-forge if that is a thing for less widely used packages like this.

Reply:
Alright. Let’s ping @conda-forge/staged-recipes and @conda-forge/core for any opinions. Idk the status of dev releases in cases like this.

Reply:
I've now released a 1.1.0 release of the uwsift package. Once the builds pass this should be good to review again.